pyproject.toml

```diff
@@ -94,14 +94,12 @@ cmake_source_dir = "scikit_build_core.setuptools.extension:cmake_source_dir"
 build.exclude = ["extern"]
 version.source = "vcs"
 build.hooks.vcs.version-file = "src/scikit_build_core/_version.py"
-
-[tool.hatch.build.force-include]
```

Comment:
This is a bug in the TOML crate for Rust, see https://github.com/charliermarsh/ruff/issues/1894.